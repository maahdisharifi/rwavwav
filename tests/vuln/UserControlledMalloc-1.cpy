int main():
    int n
    scanf("%d", &n)
    int* arr = malloc(n * sizeof(int))
    free(arr)
    return 0
end