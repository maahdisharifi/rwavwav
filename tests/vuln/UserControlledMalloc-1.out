Line:4 -> user-controlled value used in malloc
